decidim-core/app/assets/stylesheets/decidim/extras/_callout.scss

```diff
@@ -1,5 +0,0 @@
-main {
-  .callout {
-    margin: 0;
-  }
-}
```

Comment:
This change seems to have reintroduced #1550, which was fixed with this CSS in #1603. Was this just a cleanup or was it necessary for this PR?

Reply:
Oh I'm sorry! I think we should have been more specific about this CSS selector. If we remove margins from all callouts, then other ugly things happen in other places xD. Let me see if we can find a way to fix this.

Sorry!

Reply:
Totally agreed, no problem! :)